lib/spack/spack/environment.py

```diff
@@ -514,6 +515,7 @@ def __init__(self, path, init_file=None, with_view=None):
                 path to the view.
         """
         self.path = os.path.abspath(path)
+        self.coconcretize = False
```

Comment:
Can you change this variable to `concretize_together` to match the rest of the PR?

Reply:
Done